Library/Homebrew/test/cask/cask_loader/from_api_loader_spec.rb

```diff
@@ -62,100 +62,92 @@
           FileUtils.rm_rf foo_tap.path
         end
 
-        it "returns the core cask if the short name clashes with a tap migration rename" do
-          (foo_tap.path/"tap_migrations.json").write <<~JSON
-            { "#{token}": "homebrew/cask/#{token}-v2" }
-          JSON
-
-          expect(Cask::CaskLoader::FromNameLoader.try_new(token))
-            .to be_a(Cask::CaskLoader::FromNameLoader)
-            .and have_attributes(token:)
-        end
```

Comment:
This test is not so relevant anymore. With the API, short names will be evaluated before ever hitting the name loader.